public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -57,6 +57,7 @@ type Props = OwnProps & ConnectedProps<typeof connector>;
 class DashNav extends PureComponent<Props> {
   constructor(props: Props) {
     super(props);
+    this.props.dashboard.meta.canShare = false;
```

Comment:
In instances where we make changes that we either aren't upstreaming in the short term, or don't think we can ever upstream, do we think it would be valuable to include a comment above the change identifying it as an NI customization? I wonder if that would make it easier to identify our changes when we're dealing with merge conflicts while upgrading to newer versions of Grafana.

Reply:
Hmm. My first thought is that a comment for deleted code might be a little silly, so perhaps we don't do anything in that case.

Reply:
Comments added.